frontend/src/interfaces/IPropertyApiModel.ts

```diff
@@ -1,42 +1,51 @@
-import { Api_Address } from './Address';
-import { Api_ConcurrentVersion } from './ConcurrentVersion';
-import Api_TypeCode from './TypeCode';
+import { Api_Address } from 'models/api/Address';
+import { Api_ConcurrentVersion } from 'models/api/ConcurrentVersion';
+import Api_TypeCode from 'models/api/TypeCode';
 
-export interface Api_Property extends Api_ConcurrentVersion {
+import { ILease } from './ILease';
+
+/**
+ * Pairs with the current implementation of Pims.Api.Areas.Property.Models.Property.PropertyModel
+ */
+export interface IPropertyApiModel extends Api_ConcurrentVersion {
```

Comment:
Aligned the model with the model that is coming back from the backend.

Reply:
I like your refactoring and personally have no preference on what we call the frontend models but didn't we agree to start naming things API_* ? happy to hear your thoughts

Reply:
NVM, your comment on another file answered why we are not using API_Property here